src/systems/lift_drag/LiftDrag.cc

```diff
@@ -259,6 +265,11 @@ void LiftDragPrivate::Update(EntityComponentManager &_ecm)
   const auto worldPose =
       _ecm.Component<components::WorldPose>(this->linkEntity);
 
+  // get wind as a component from the _ecm
+  Entity windEntity = _ecm.EntityByComponents(components::Wind());
+  auto windLinearVel =
```

Comment:
What if there is no wind component? This is very common as many of the worlds will not define a wind component. Check if `windLinearVel` ia a nullptr before using it.